engine/src/exec-interface.cpp

```diff
@@ -4290,12 +4290,9 @@ bool MCInterfaceExecSortContainer(MCExecContext &ctxt, MCStringRef p_data, int p
 	                             kMCStringOptionCompareExact))
 		return false;
 
+	extern bool MCStringsSplit(MCStringRef p_string, MCStringRef p_separator, MCStringRef*&r_strings, uindex_t& r_count);
```

Comment:
The reason it's not is that it's implemented in `exec-strings.cpp`, but it shouldn't be from the point of view of the modularised exec functionality. I believe it was deemed too high-level to be worthy of inclusion in `foundation-string`.